### PR TITLE
23-ui-new-position-view

### PR DESCRIPTION
**Pull request by zwypych**
change button's names into eng

### CODE REVIEW

File: src/views/NewPosition/NewPosition.styles.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+export const StyledNewPosition = styled.section`
+  margin: 10px 30px;
```
Comment (michakow) [inline src/views/NewPosition/NewPosition.styles.js:3]: margin causes shift to right side

File: src/App.js [hunk 2]
```diff
@@ -26,35 +13,14 @@ function App() {
...
         <Routes>
-          <Route path="*" element={<NotFound />} />
+          {/* <Route path="*" element={<NotFound />} />
 
           <Route
             path="/"
             element={<h1 className="text-6xl font-bold p-2">Expenses App</h1>}
-          />
+          /> */}
+          <Route path="*" element={<NewPosition />} />
```
Comment (michakow) [inline src/App.js:22]: uncomment above code and use another path, please

Comment (zwypych) [reply src/App.js:22]: corrected

File: src/views/NewPosition/NewPosition.styles.js (hunk 1 above)
Comment (zwypych) [reply src/views/NewPosition/NewPosition.styles.js]: deleted

**Pull request merged**